internal/controllers/utils/utils_test.go

```
@@ -1675,3 +1677,28 @@ var _ = Describe("Server predicate functions", func() {
+	})
+
+	It("should set PFS cipher suites on GetClientTLSConfig", func() {
+		ctx, cancel := context.WithCancel(context.Background())
+		defer cancel()
+
+		cfg, err := GetClientTLSConfig(ctx, "", "", "")
+		Expect(err).NotTo(HaveOccurred())
+		Expect(cfg.CipherSuites).To(Equal(pfsCipherSuites))
+	})
+})
+
```

golangci-lint (2.12.2)

[error] File is not properly formatted

(gofmt)